src/screens/authentication/components/Auth/ForgotPassword/styles.js

```diff
@@ -0,0 +1,19 @@
+import { StyleSheet } from 'react-native'
+import { scale, verticalScale } from 'react-native-size-matters'
+
+const styles = StyleSheet.create({
+  container: {
+    marginBottom: verticalScale(37)
+  },
+  text: {
+    textAlign: 'center',
+    fontFamily: 'Open Sans',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: scale(14),
+    lineHeight: verticalScale(14),
+    color: '#FE9458'
+  }
+})
+
+export default styles
```

Comment:
Use `react-native-size-matters/extend`

And remove the custom text properties (fontFamiliy, fontStyle, fontWeight, color). Use the theme properties instead